scripts/oqsprovider-pkcs12gen.sh

```diff
@@ -47,7 +52,7 @@ if [ $? -ne 0 ] || [ ! -f tmp/$1_srv_1.p12 ]; then
 fi
 
 # Generate config file with oqsprovider disabled
-sed -e 's/^oqsprovider/# oqsprovider/' "$(pwd)/scripts/openssl-ca.cnf" > tmp/openssl-ca-no-oqsprovider.cnf
+sed -e 's/^oqsprovider/# oqsprovider/' $OPENSSL_CONF > tmp/openssl-ca-no-oqsprovider.cnf
```

Comment:
```suggestion
sed -e 's/^oqsprovider/# oqsprovider/' "$OPENSSL_CONF" > tmp/openssl-ca-no-oqsprovider.cnf
```
Wouldn't hurt to quote this in case somebody has a space somewhere in the filepath.